src/core/userDataHelper.js

```diff
@@ -30,6 +33,35 @@ var userDataHelper = {
       return userId === team.userid;
     });
   },
+  /**
+   * Get the list of tagged words for a user's notes. Memoize result.
+   * @param userid
+   * @param notes
+   * @returns {Array}
+   */
+  getWordbankWords: function(userid, notes) {
```

Comment:
Could you provide more info on what you mean by "reside in the api wrapper"? Do you mean something like an `api.user.getWordbank` function in `tidepool_platform.js`?